mftf/2.3/configuration.md

```diff
@@ -156,8 +164,8 @@ MAGENTO_RESTAPI_SERVER_PORT=5000
 
 ### TESTS
 
-These values can be used in cases where you are working locally on both MFTF's implementation code and Magento's test case code.
-Use them if you have a more advanced local development setup that involves symlinking the MFTF into the `vendor` directory of the Magento root directory.
+These values can be used in cases where you are working locally on both MFTF implementation code and Magento test case code.
+Use them if you have a more advanced local development setup that involves using symbolic links for the MFTF into the `vendor` directory of the Magento root directory.
```

Comment:
Changes "involves using symbolic links for the MFTF" to "symbolically links the MFTF".

(Actually, I prefer symlinks.)

Reply:
Will change to:
Overrides base paths for the framework.
You may need it when the MFTF is applied as a separate tool.
For example, when you contribute to the MFTF and want to place the testing framework and the Magento code base in separate projects.